tests/hats/catalog/dataset/test_collection_properties.py

```diff
@@ -84,6 +84,29 @@ def test_read_collection_list_parse(tmp_path):
     ]
 
 
+def test_write_properties_colon_not_escaped(tmp_path):
+    test_timestamp_value = "2025-06-30T17:10UTC"
+
+    collection_props = CollectionProperties(
+        name="test_collection_with_colon",
+        hats_primary_table_url="test_url_with_colon",
+        obs_regime=test_timestamp_value,
```

Comment:
I know that technically it doesn't matter, but it really confused me that `obs_regime` is a date
```suggestion
        hats_creation_date=test_timestamp_value,
```